### ISSUE TITLE
Outdir is relative to package.json, rather than the file

### ISSUE THREAD

**Issue by shankiesan** [mikestead/openapi-client#15]
When I use openapi.genCode(), it outputs the file relative to package.json, rather than the file in which it was called. Is this the expected behaviour? It's a bit different from what most packages do.

Otherwise, great package! Love it!

**Comment by mikestead** [comment]
@shankiesan yeah this is a valid point. It will be a breaking change when done so will require a major version bump. I'll squeeze it in next time I'm in the code.

**Comment by shankiesan** [comment]
Yeah fair enough. No major rush – maybe just add it to the docs for the time being?
